### DISCUSSION TITLE
Tabs and tables together. Request for help.

### DISCUSSION THREAD

**Discussion by monointeractive** [filamentphp/filament#10068] [General]
Please help.
I have an application that does not use Panel Builder. Uses tables and forms as livewire components.

Is there any way for me to add infolist tabs above the table? This topic is very urgent for me. Please advise.

**Comment by monointeractive** [comment]
I got help on the Discord server:
https://discord.com/channels/883083792112300104/1181239883965157467

**Comment by sbc640964** [comment]
@monointeractive I would be happy if you present the solution here
